test_post_upload_large_image_jpg failure in system_tests_http2

```

=========================================================================== FAILURES ===========================================================================
_____________________________________________________ Http2TestTwoRouter.test_post_upload_large_image_jpg ______________________________________________________

self = <tests.system_tests_http2.Http2TestTwoRouter testMethod=test_post_upload_large_image_jpg>

    @unittest.skipIf(skip_test(), "Python 3.7 or greater, Quart 0.13.0 or greater and curl needed to run http2 tests")
    @unittest.skipIf(skip_test_in_ci('QPID_SYSTEM_TEST_SKIP_HTTP2_LARGE_IMAGE_UPLOAD_TEST'), skip_reason)
    def test_post_upload_large_image_jpg(self):
        # curl  -X POST -H "Content-Type: multipart/form-data"  -F "data=@/home/gmurthy/opensource/test.jpg"
        # http://127.0.0.1:9000/upload --http2-prior-knowledge
        address = self.get_address() + "/upload"
        _, out, _ = self.run_curl(address, args=self.get_all_curl_args(['-X', 'POST', '-H',
                                                                        'Content-Type: multipart/form-data',
                                                                        '-F', 'data=@' + image_file('test.jpg')]))
>       self.assertIn('Success', out)
E       AssertionError: 'Success' not found in '<!doctype html>\n<html lang=en>\n<title>408 Request Timeout</title>\n<h1>Request Timeout</h1>\n<p>The server closed the network connection because the browser didn&#39;t finish the request within the specified time.</p>\n'

/home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_tests_http2.py:191: AssertionError
---------------------- generated xml file: /__w/skupper-router/skupper-router/skupper-router/build/tests/junitxmls/system_tests_http2.xml ----------------------
=================================================================== short test summary info ====================================================================
FAILED ::Http2TestTwoRouter::test_post_upload_large_image_jpg - AssertionError: 'Success' not found in '<!doctype html>\n<html lang=en>\n<title>408 Request Timeout</title>\n<h1>Request Timeout</h1>\n<p>The server closed the network connection because the browser didn&#39;t finish the request within the specified time.</p>\n'

```

https://github.com/skupperproject/skupper-router/actions/runs/5426643610/jobs/9868956398?pr=1147#step:34:3389

--- a/tests/system_tests_http2.py
+++ b/tests/system_tests_http2.py
@@ -179,16 +179,16 @@ class CommonHttp2Tests:
 
     skip_reason = 'Test skipped on certain Travis environments'
 
-    @unittest.skipIf(skip_test(), "Python 3.7 or greater, Quart 0.13.0 or greater and curl needed to run http2 tests")
-    @unittest.skipIf(skip_test_in_ci('QPID_SYSTEM_TEST_SKIP_HTTP2_LARGE_IMAGE_UPLOAD_TEST'), skip_reason)
-    def test_post_upload_large_image_jpg(self):
+#    @unittest.skipIf(skip_test(), "Python 3.7 or greater, Quart 0.13.0 or greater and curl needed to run http2 tests")
+#    @unittest.skipIf(skip_test_in_ci('QPID_SYSTEM_TEST_SKIP_HTTP2_LARGE_IMAGE_UPLOAD_TEST'), skip_reason)
+#    def test_post_upload_large_image_jpg(self):
         # curl  -X POST -H "Content-Type: multipart/form-data"  -F "data=@/home/gmurthy/opensource/test.jpg"
         # http://127.0.0.1:9000/upload --http2-prior-knowledge
-        address = self.get_address() + "/upload"
-        _, out, _ = self.run_curl(address, args=self.get_all_curl_args(['-X', 'POST', '-H',
-                                                                        'Content-Type: multipart/form-data',
-                                                                        '-F', 'data=@' + image_file('test.jpg')]))
-        self.assertIn('Success', out)
+#        address = self.get_address() + "/upload"
+#        _, out, _ = self.run_curl(address, args=self.get_all_curl_args(['-X', 'POST', '-H',
+#                                                                       'Content-Type: multipart/form-data',
+#                                                                        '-F', 'data=@' + image_file('test.jpg')]))
+#        self.assertIn('Success', out)
 
     @unittest.skipIf(skip_test(), "Python 3.7 or greater, Quart 0.13.0 or greater and curl needed to run http2 tests")
     def test_delete_request(self):

